dot/rpc/subscription/websocket_test.go

```diff
@@ -229,8 +229,15 @@ func TestWSConn_HandleConn(t *testing.T) {
 	require.Equal(t, `{"jsonrpc":"2.0","result":8,"id":0}`+"\n", string(msg))
 
 	// test initExtrinsicWatch with invalid transaction
-	coreAPI := mocks.NewCoreAPI(t)
-	coreAPI.On("HandleSubmittedExtrinsic", mock.AnythingOfType("types.Extrinsic")).Return(runtime.ErrInvalidTransaction)
+	transactionValidityErr := errors.NewTransactionValidityError()
+	invalidTransaction := errors.NewInvalidTransaction()
+	err = invalidTransaction.Set(errors.Future{})
+	errMsg := fmt.Errorf("%w: %s", errors.ErrInvalidTxn, transactionValidityErr.Error())
```

Comment:
nit this **should** *work*

```suggestion
	errMsg := fmt.Errorf("%w: %s", errors.ErrInvalidTxn, transactionValidityErr)
```

Reply:
It did not, not of type string for %s

Reply:
Fixed!

Reply:
Ohhh didn't think of that one! Thanks Tim!